app/models/base.rb

```diff
@@ -0,0 +1,18 @@
+class Base
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  #extend ActiveModel::Naming
+
+  def self.step_name
+    name.underscore
+  end
+
+  def step_name
```

Comment:
I can't see value in maintaining this as an instance method as well as a class method?

Reply:
Me neither